src/NServiceBus.RabbitMQ/Configuration/RabbitMQTransportSettingsExtensions.cs

```diff
@@ -64,5 +64,17 @@ public static TransportExtensions<RabbitMQTransport> TimeToWaitBeforeTriggeringC
             transportExtensions.GetSettings().Set(SettingsKeys.TimeToWaitBeforeTriggeringCircuitBreaker, waitTime);
             return transportExtensions;
         }
+
+        /// <summary>
+        /// Overrides the default usePublisherConfirms or value set in connection string.
+        /// </summary>
+        /// <param name="transportExtensions"></param>
+        /// <param name="publisherConfirms">Controls if publisher confirms should be used.</param>
+        /// <returns></returns>
+        public static TransportExtensions<RabbitMQTransport> UsePublisherConfirms(this TransportExtensions<RabbitMQTransport> transportExtensions, bool publisherConfirms)
```

Comment:
@adamralph I like this question a lot. 
I spent few minutes looking for precedents in other transports and below are my findings:
I found 2 examples of extension method without a parameter and 1 with :)
 
 - ASQ without parameter:
https://github.com/Particular/NServiceBus.AzureStorageQueues/blob/release-7.0.0/src/Transport/Config/AzureStorageTransportExtensions.cs#L75-L79
 - ASB without parameter:
https://github.com/Particular/NServiceBus.AzureServiceBus/blob/develop/src/Transport/Config/ExtensionMethods/AzureServiceBusTransportExtensions.cs#L100-L104
 - ASB with parameter:
https://github.com/Particular/NServiceBus.AzureServiceBus/blob/develop/src/Transport/Config/ExtensionMethods/AzureServiceBusTransportExtensions.cs#L55-L59

My take is. At the moment default value is true. However as default value can change (even that in this case it is highly unlikely), putting UsePublisherConfirms(true) keep you on the safe side. 

What is more to make this method useful without parameter we would have to negate it to something like: DisablePublisherConfirms(). I would put my vote for leaving it with value.